### DISCUSSION TITLE
Cannot find wireguard in the add-on store

### DISCUSSION THREAD

**Discussion by lutterodt** [hassio-addons/addon-wireguard#132] [Q&A]
I run hassio as the main OS on a raspberry pi 4B. I have searched for wireguard and other add-ons (VSC amongst others) but I'm only limited to a few "official" add-ons. I think I'm doing something wrong. Can anybody help me out?

**Comment by frenck** [comment]
It should be there by default, if it isn't you can add the repository manually.

Use the following repository URL:

`https://github.com/hassio-addons/repository`

**Reply by lutterodt** [reply]
Fantastic! It worked. Thanks Frenck!
